.buildkite/pipeline.gpu.large.yml

```diff
@@ -36,6 +36,6 @@
   conditions: ["RAY_CI_ML_AFFECTED"]
   commands:
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/travis/upload_build_info.sh; fi }; trap cleanup EXIT
-    - DATA_PROCESSING_TESTING=1 TRAIN_TESTING=1 TUNE_TESTING=1 ./ci/travis/install-dependencies.sh
+    - DATA_PROCESSING_TESTING=1 TRAIN_TESTING=1 TUNE_TESTING=1 PYTHON=3.7 ./ci/travis/install-dependencies.sh
```

Comment:
For my learning, is this needed?

Reply:
Torch geometric does not support python 3.6.

We could just make a separate build just for 3.7, but I thought it would be better to just upgrade everything to 3.7 since this is what we do for Tune anyways currently.

Reply:
Oh wait isn't the default value 3.7?

Reply:
No it's 3.6 I believe.

Reply:
Ah I believe it was updated for GPU images [here](https://github.com/ray-project/ray/pull/21711)

But similar to my comment on that PR, having it explicit makes sense (in case we change default version in the future)

Reply:
Ah got it. Actually there are versions of torch geometric that support python 3.6, but the later versions don't. But in any case, it's fine to have this be explicit?